tests/flow/test_index_create.py

```diff
@@ -271,3 +285,493 @@ def test07_index_creation_undefined_identifier(self):
             self.env.assertTrue(False)
         except ResponseError as e:
             self.env.assertContains("a not defined", str(e))
+
+    def test08_async_index_creation(self):
+        # 1. create a large graph
```

Comment:
when index operation is replicated do we need to wait for the index to finish population before we run write queries?

Reply:
I don't believe so.

1. `CREATE INDEX` received on primary, the index object is created visible only to WRITE queries (for index updates) and the indexer. All of this is done on our writer thread.
2. primary replicates the command to the replica
3. Replica performs step (1)
4. write operations received on primary (including index operations) are replicated in order.

So I believe we're OK.